src/utils.ts

```diff
@@ -53,6 +54,13 @@ export function serializeEntity<T extends IEntity>(
   subColMetadata.forEach(scm => {
     delete serializableObj[scm.propertyKey];
   });
+
```

Comment:
Can you create some tests for this case? After serialization, the "ignored" properties shouldn't be serialized